predeploy.py

```diff
@@ -21,5 +21,23 @@
 else:
     for f in pyfiles:
         shutil.copyfile(os.path.join(app_path, f), f)
+    lines = []
+    ga_line = 0
+    with open(os.path.join(app_path, "app.py"), "r") as f:
+        lines = f.readlines()
```

Comment:
you should be able to move the next block into here and go line by line without slurping it all into memory, by using the iterator.